src/models/MarkdownFile.ts

```diff
@@ -56,33 +56,30 @@ export class MarkdownFile {
 
 
   private extractMarkedText(content: string): string {
-    // extract the text between markers - markdown comments <!-- BEGIN_ANKI_CARDS --> text <!-- END_ANKI_CARDS -->
-    // if the markers are not present then return the entire content
+    // Extract the text between markers - markdown comments <!-- BEGIN_ANKI_CARDS --> text <!-- END_ANKI_CARDS -->
```

Comment:
Could you move these comments above the function using /** */ syntax? 
its just so then you would get the function description when hovering over the symbol